_includes/feed.liquid

```diff
@@ -99,15 +99,11 @@ section: ethereum
 </style>
 
 {% case metadata.l2healthflag %}
-  {% when "arbitrum" %}
+  {% when "arbitrum" or "optimism" or "metis" %}
```

Comment:
I see the callout only on Arbitrum: https://documentation-git-fork-dwightjl-l2-sequenc-c2624d-chainlinklabs.vercel.app/docs/arbitrum-price-feeds/


but not seeing it on Optimism and metis :
https://documentation-git-fork-dwightjl-l2-sequenc-c2624d-chainlinklabs.vercel.app/docs/optimism-price-feeds/
https://documentation-git-fork-dwightjl-l2-sequenc-c2624d-chainlinklabs.vercel.app/docs/data-feeds-metis/

Is this the expected behaviour ?

Reply:
Good catch. I switched the logic from liquid cases to a liquid if statement.